src/application/shell.ts

```diff
@@ -97,7 +97,16 @@ class ApplicationShell extends Widget {
 
     let topPanel = this._topPanel = new Panel();
     let hboxPanel = this._hboxPanel = new BoxPanel();
-    let dockPanel = this._dockPanel = new DockPanel();
+    let dockPanel = this._dockPanel = new DockPanel({
+      renderer: {
+        createTabBar: () => {
+          let bar = new TabBar();
+          bar.addClass('jp-mod-top');
```

Comment:
There are styles that @ellisonbg wrote for `p-mod-top` and `p-mod-bottom` when they both existed and I think the idea was to have both types available in the UI, but that we were only using the top ones for now. So instead of deleting those selectors in the stylesheet, I opted to add them in our custom `shell.ts`.